runOnOpenShift.sh

```diff
@@ -163,7 +163,7 @@ echo
 echo -e "$summary"
 echo
 
-declare -l answer_continue # -l converts the value to lower case before it's assigned
+declare answer_continue # -l converts the value to lower case before it's assigned
```

Comment:
Misleading comment;
Might be able to use `shopt` to duplicate old behavior for "Y" https://stackoverflow.com/a/1728814

Reply:
Good catch, I'll remove the comment.

I've decided to no longer support the case insensitive response processing. If it requires anything more complex than the `-l` switch, it's not worth it.